start_benchmark.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PHP_COMMAND="php -n ${PHP_OPTIONS}"
+NODEJS_COMMAND='node'
+
+AERYS_COMMAND="./aerys/vendor/bin/aerys -w 1 -c ./aerys/server.php --worker-args=\"-n\""
```

Comment:
`-n` doesn't make sense for `--worker-args` if you pass a config just after that.